sfge/src/test/java/com/salesforce/rules/fls/apex/operations/FlsViolationUtilsTest.java

```diff
@@ -76,7 +77,17 @@ public void testMessageWithRelationalField() {
         assertThat(
                 message,
                 equalToIgnoringCase(
-                        "FLS validation is missing for [UPDATE] operation on [My_Obj__c] with field(s) [Name,Status__c] - SFGE may not have parsed some objects/fields correctly. Please confirm if the objects/fields involved in these segments have FLS checks: [Relational_Field__r.Another_field__c]"));
+                        String.format(
```

Comment:
Modified failing tests to use String formats instead of hardcoding the message. Will clean up the remaining tests at another iteration while applying Teresa's feedback.